ordered_associative_task/student/include/impl.h

```diff
@@ -15,4 +15,20 @@
  *
  * @return 1-5
  */
-size_t convertMarkTo5PointSystem(size_t markIn100Point);
+size_t convertMarkTo5PointSystem(size_t markIn100Point) {
+    std::map<int, int> pointSystem {
+        {29, 1},
+        {49, 2},
+        {69, 3},
+        {89, 4},
+        {90, 5}
+    };
+
+    auto reultIter = pointSystem.lower_bound(markIn100Point);
+
+    if (reultIter == pointSystem.cend()) {
```

Comment:
try to avoid this check at all
hint: use upper_bound

Reply:
done